genai/tools/tools-code-exec-with-txt.js

```diff
@@ -20,17 +20,17 @@ const {GoogleGenAI} = require('@google/genai');
 const GOOGLE_CLOUD_PROJECT = process.env.GOOGLE_CLOUD_PROJECT;
 const GOOGLE_CLOUD_LOCATION = process.env.GOOGLE_CLOUD_LOCATION || 'global';
 
-async function generateContent(
+async function generateAndExecuteCode(
   projectId = GOOGLE_CLOUD_PROJECT,
   location = GOOGLE_CLOUD_LOCATION
 ) {
-  const ai = new GoogleGenAI({
+  const client = new GoogleGenAI({
     vertexai: true,
     project: projectId,
     location: location,
   });
 
-  const response = await ai.models.generateContent({
+  const response = await client.models.generateContent({
     model: 'gemini-2.5-flash',
     contents:
       'What is the sum of the first 50 prime numbers? Generate and run code for the calculation, and make sure you get all 50.',
```

Comment:
@Guiners: Can you point me to the location where your found this prompt ?

I see a different one at https://github.com/GoogleCloudPlatform/python-docs-samples/blob/0ab9566fc069b7769096d2292fa3638b34da33af/genai/tools/tools_code_exec_with_txt.py#L32